tmx_object.go

```diff
@@ -167,8 +167,8 @@ type Text struct {
 	Size int `xml:"pixelsize,attr"`
 	// Whether word wrapping is enabled (1) or disabled (0). Defaults to 0.
 	Wrap bool `xml:"wrap,attr"`
-	// Color of the text in #AARRGGBB or #RRGGBB format (default: #000000)
-	Color string `xml:"color,attr"`
+	// Color of the text, Default nil
```

Comment:
Please update color that default is #000000

Reply:
done